pkg/server/commands/reverseexpand/reverse_expand.go

```diff
@@ -129,6 +131,12 @@ func WithResolveNodeLimit(limit uint32) ReverseExpandQueryOption {
 	}
 }
 
+func WithDispatchThrottler(dispatchThrottler *throttler.DispatchThrottler) ReverseExpandQueryOption {
+	return func(d *ReverseExpandQuery) {
+		d.dispatchThrottler = dispatchThrottler
```

Comment:
Please create a `NoOpDispatchThrottler` and set is as the default in the constructor, then you can get rid of the nil checks in lines 345 and 553